arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist

```diff
@@ -312,7 +312,7 @@
 				<key>displayName</key>
 				<string>Show device location using indoor positioning</string>
 				<key>descriptionText</key>
-				<string>Show your device's real-time location while inside a building by using signals from indoor positioning beacons.</string>
+				<string>Show your device&apos;s real-time location while inside a building by using signals from indoor positioning beacons.</string>
```

Comment:
Hmm, weird, I believe this is already on `v.next`. Try to rebase or merge v.next to your branch to keep up to date.